bitswap/src/behaviour.rs

```diff
@@ -86,7 +86,7 @@ impl Stats {
 pub struct Bitswap {
     /// Queue of events to report to the user.
     events: VecDeque<NetworkBehaviourAction<Message, BitswapEvent>>,
-    /// List of peers to send messages to.
+    /// List of prospect peers to connect to.
```

Comment:
Does this mean the peers which bitswap has dialed for or what exactly? I am pretty sure it's not the "send messages to" nor has it been for a long time :)

Reply:
Yeah, I'm wondering if this object shouldn't be removed entirely, as it is only applied in connection attempts from `Bitswap`, and not used for anything else as far as I can tell.

Reply:
Ok, it does makes some sense, as it limits the number of `DialPeer` attempts. Still needs to be removed, but that's material for a follow-up.